js/classes/entities/midground/enemies/snowy/yeti.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Class that represents a Yeti.
+ * 
+ * @author Trae Claar
+ */
+class Yeti {
+    /**
+     * Constructor for a Yeti.
+     * 
+     * @param {Vector} pos the initial position of the Yeti
+     */
+    constructor(pos) {
+        this.base = new EnemyBase(
+            this, 
+            pos, 
+            Yeti.SCALED_SIZE, 
+            Yeti.SPEED, 
+            Yeti.MAX_HEALTH, 
+            Yeti.ROAM_DISTANCE, 
+            () => this.handleDeath(),
+            EnemyBase.AGGRESSIVE_STANCE
+        );
+
+        this.animations = [];
+        this.loadAnimations();
+
+        this.lastAttack = 0;
+        this.dealtDamage = false;
+    };
+
+    /** The size, in pixels, of the Yeti on its spritesheet. */
+    static get SIZE() {
+        return new Vector(64, 64);
+    }
+
+    /** The scale factor applied to the Yeti when drawing. */
+    static get SCALE() {
+        return 5;
+    };
+
+    /** The size, in pixels, of the Yeti on the canvas. */
+    static get SCALED_SIZE() {
+        return Vector.multiply(Yeti.SIZE, Yeti.SCALE);
+    }
+
+    /** The speed of the Yeti. */
+    static get SPEED() {
+        return Yeti.SCALE * 10;
+    }
+
+    /** The file path to the Yeti's spritesheet. */
+    static get SPRITESHEET() {
+        return "./sprites/yeti.png";
+    };
+
+    /** The maximum health of the Yeti. */
+    static get MAX_HEALTH() {
+        return 100;
+    };
+
+    /** The distance the Yeti will wander from its original position when roaming. */
+    static get ROAM_DISTANCE() {
+        return Yeti.SCALED_SIZE.x * 2;
+    };
+
+    /** The number of seconds between attacks. */
+    static get ATTACK_COOLDOWN() {
+        return 3;
+    };
+
+    /** The amount of damage a Yeti deals during an attack. */
+    static get ATTACK_DAMAGE() {
+        return 50;
+    };
+
+    /** Number of seconds after the start of the attack animation when damage should be dealt. */
+    static get DAMAGE_DELAY() {
+        return 0.9375;
+    }
+
+    /**
+     * Perform any necessary operations when the Yeti dies.
+     */
+    handleDeath() {
+        // if the enemy is now dead, remove it from the game
+        // replace this code with any death effects, state changes, etc.
+        this.removeFromWorld = true;
+    }
+    
+    /** Update the Yeti. */
+    update() {
+        this.base.update();
+
+        const secondsSinceLastAttack = Date.now() / 1000 - this.lastAttack;
+
+        // if we've finished our current attack, change action to idle
+        if (this.action === "attacking" 
+            && this.animations[this.base.getFacing()]["attacking"].totalTime < secondsSinceLastAttack) {
```

Comment:
Should this be a less than or a greater than?

Reply:
Why would it be a greater than?